src/Aspire.Dashboard/Configuration/DashboardOptions.cs

```diff
@@ -305,3 +306,45 @@ internal bool TryParseOptions([NotNullWhen(false)] out IEnumerable<string>? erro
         return messages is null;
     }
 }
+
+public sealed class DebugSession
```

Comment:
All the other option types end with `Options`.
```suggestion
public sealed class DebugSessionOptions
```